spec/requests/api/v1/accounts_spec.rb

```diff
@@ -163,6 +163,31 @@
       end
     end
 
+    context 'when user tries to follow their own account' do
+      subject do
+        post "/api/v1/accounts/#{other_account.id}/follow", headers: headers
+      end
+
+      let(:locked) { false }
+      let(:other_account) { user.account }
+
+      it 'returns http forbidden and error message' do
+        subject
+
+        error_msg = 'Following your own account is not allowed'
```

Comment:
This can be pulled from I18n as well, instead of repeated (ie, remove this local variable and just use the `I18n.t(...)` block in the assertion below.

Reply:
Good catch. Have modified the test accordingly 👍🏿